libs/cart/src/models/composite-cart-item.ts

```diff
@@ -6,7 +6,7 @@ export interface DaffCompositeCartItem extends DaffCartItem {
 }
 
 export interface DaffCompositeCartItemOption {
-	option_id: string;
+	option_id: string | number;
```

Comment:
You can use the `ID` type from `@daffodil/core`

Reply:
changed